system/emergency_handler/src/emergency_handler/emergency_handler_core.cpp

```diff
@@ -224,6 +224,10 @@ void EmergencyHandler::callMrmBehavior(
   auto request = std::make_shared<tier4_system_msgs::srv::OperateMrm::Request>();
   request->operate = true;
 
+  if (mrm_behavior == MrmState::NONE) {
+    RCLCPP_WARN(this->get_logger(), "MRM behavior is None. Do nothing.");
```

Comment:
No need to use THROTTLE?

Reply:
I don't think it is necessary because it is not the kind of warning that would display continuously.
Also, the timing of each warning is tied to the vehicle behavior, so I would like to display the warnings as they occer.

If they display continuously at unintended timing, I will change it to use THROTTLE.

Reply:
Okay, thank you!

Reply:
I agree it is not too much output.
I previously watched `invalid MRM behavior` which may be nearly same frequency as that warning, and it looks adequate amount.